packages/react-dom/src/client/ReactDOMFiberTextarea.js

```diff
@@ -11,6 +11,7 @@ import invariant from 'shared/invariant';
 import warning from 'shared/warning';
 
 import ReactControlledValuePropTypes from '../shared/ReactControlledValuePropTypes';
+import getSafeValue from './getSafeValue';
 
 let didWarnValDefaultVal = false;
 
```

Comment:
The type of initialValue in the _wrapperState below should probably be `SafeValue` now, just like we did for input elements.

Reply:
Just pushed my changes - I hope they make sense and I took the right approach :-)